_posts/2023-06-15-registration-for-opensearchcon-2023-is-now-open.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2023-06-15T22:09:26.908Z
+excerpt: The OpenSearch Project invites all builders to Seattle this fall, to
+  celebrate the success of the OpenSearch Project and look to the future of
+  search, analytics, and Generative AI. Join us to learn how to build at scale!
+meta_description: OpenSearchCon 2023 - Coming to the Sheraton Grand Hotel in
+  Seattle September 27-29. Register today!
+meta_keywords: opensearchcon, opensearchcon 2023, register for opensearchcon,
+  opensearchcon registration
+feature_image: /assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg
+authors:
+  - daryllswager
+layout: post
+title: Registration for OpenSearchCon 2023 is now open!
+categories:
+  - community
+---
+![opensearchcon 2022 stage](/assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg "opensearchcon crowd")
```

Comment:
Suggest changing "opensearchcon crowd" to "Laysa Uchoa giving her keynote speech at OpenSearconCon 2022" 

need to have {: .img-fluid}   after the last  )